api/nakadi-event-bus-api.yaml

```diff
@@ -1126,18 +1134,6 @@ definitions:
             `metadata` and any additionally defined properties passed in the `EventTypeSchema` directly on
             top level of the Event. If name conflicts arise, creation of this EventType will be rejected.
             Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
-      effective_schema:
```

Comment:
There are some references to this field spread in this file. I think we should search for then and remove as well.